includes/ocos.h

```diff
@@ -3,19 +3,21 @@
 
 #pragma once
 
-#include <vector>
+#include <algorithm>
 #include <functional>
+#include <iterator>
+#include <vector>
 
 #define ORT_API_MANUAL_INIT
 #include "onnxruntime_cxx_api.h"
 #undef ORT_API_MANUAL_INIT
 
-
 // A helper API to support test kernels.
 // Must be invoked before RegisterCustomOps.
 extern "C" bool ORT_API_CALL AddExternalCustomOp(const OrtCustomOp* c_op);
 
-const char c_OpDomain[] = "ai.onnx.contrib";
+constexpr const char* c_OpDomain = "ai.onnx.contrib";
+constexpr const char* c_ComMsExtOpDomain = "com.microsoft.extensions";
```

Comment:
This domain name looks better but I don't know the way we can migrate all the user's existing model to this new name?  Do you have a solution for this? Or we move this change to another PR to address the issue completely?

Reply:
I'm only using the new domain name for the new ops currently. For additional things we want to include in the packages (e.g. BERT tokenizer) we can register in both domains so existing models still work. 

Does that approach work for you?

Reply:
Is there documentation somewhere on current usage of ops by partner to have an idea of who's using what? 

A script to update the domain in an existing model is trivial so easily added.